package.json

```
@@ -17,7 +21,7 @@
     "docker:build": "docker build -t opentermsarchive.org . --build-arg ENV_FILE=.env",
     "docker:run": "(docker rm opentermsarchive.org || true) && docker run --rm -p 3000:3000 --name opentermsarchive.org -it opentermsarchive.org",
     "docker:ssh": "docker exec -it opentermsarchive.org /bin/sh",
-    "test": "jest",
+    "test:links": "linkinator http://localhost:$npm_package_config_port --recurse --skip https://www.linkedin.com/.* --verbosity error",
```

On my machine, this fails. I had to use the `0.0.0.0` loopback meta-address instead of the `localhost` hostname.
I assume this is because running `npm start` states:
```
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
```
and my [hosts file](https://github.com/MattiSG/adblock/blob/master/hosts.default#L7) is set to resolve `localhost` to `127.0.0.1`.
```suggestion
    "test:links": "linkinator http://0.0.0.0:$npm_package_config_port --recurse --skip https://www.linkedin.com/.* --verbosity error",
```